tests/components/sleepiq/test_init.py

```diff
@@ -10,21 +10,25 @@
 from tests.common import load_fixture, get_test_home_assistant
 
 
-def mock_responses(mock):
+def mock_responses(mock, single=False):
     """Mock responses for SleepIQ."""
-    base_url = 'https://api.sleepiq.sleepnumber.com/rest/'
+    base_url = 'https://prod-api.sleepiq.sleepnumber.com/rest/'
+    if single:
+        suffix = '-single'
+    else:
+        suffix = ''
     mock.put(
         base_url + 'login',
         text=load_fixture('sleepiq-login.json'))
     mock.get(
         base_url + 'bed?_k=0987',
-        text=load_fixture('sleepiq-bed.json'))
+        text=load_fixture('sleepiq-bed%s.json' % suffix))
     mock.get(
         base_url + 'sleeper?_k=0987',
         text=load_fixture('sleepiq-sleeper.json'))
     mock.get(
         base_url + 'bed/familyStatus?_k=0987',
-        text=load_fixture('sleepiq-familystatus.json'))
+        text=load_fixture('sleepiq-familystatus%s.json' % suffix))
```

Comment:
use formated strings

Reply:
You mean like `'sleepiq-familystatus{}.json'.format(suffix)`? Easy enough change to make if that's it

Reply:
Or do you mean formatted string literals? Its not clear from the python3 docs which is generally preferred, but it seems to be the newest of the ways to format strings I can find.
https://docs.python.org/3/reference/lexical_analysis.html#f-strings

Reply:
Flake8 seems to not like that. Maybe too new? I'll roll back to my first guess what you meant